frontend/ui/elements/reader_menu_order.lua

```diff
@@ -163,6 +163,7 @@ local order = {
         "terminal",
         "----------------------------",
         "plugin_management",
+        "notifications",
```

Comment:
I'd rather see this burried down in menu we don't use often, like `Gear>Screen>` (it's mostly of interest with eInk devices) or `Gear>Device>`.